cost-analyzer/values-thanos.yaml

```diff
@@ -51,11 +51,15 @@ prometheus:
 thanos:
   store:
     enabled: true
+    grpcSeriesMaxConcurrency: 1
+    blockSyncConcurrency: 1
     resources: 
       requests:
-        memory: "2Gi"
+        memory: "2.5Gi"
   query: 
     enabled: true
+    timeout: 3m
+    maxConcurrent: 10
```

Comment:
Let the timeout be 3m for longer running queries from `thanos-query` -> `thanos-store` to ensure that we can appropriately queue using less concurrent request handling.